src/Makefile.am

```diff
@@ -62,6 +62,6 @@ v4l2_request_drv_video_la_LDFLAGS = -module -avoid-version -no-undefined \
 				    -Wl,--no-undefined
 v4l2_request_drv_video_la_LIBADD = $(DRM_LIBS) $(LIBVA_LIBS)
 v4l2_request_drv_video_la_LTLIBRARIES = v4l2_request_drv_video.la
-v4l2_request_drv_video_ladir = /usr/lib/dri/
+v4l2_request_drv_video_ladir = $(exec_prefix)$(DRM_LIBDIR)/dri/
```

Comment:
Hi,
You're right that this shouldn't be hardcoded, but it looks like the right path is obtained with `pkg-config libva --variable driverdir` and shouldn't be related to libdrm in particular. We should also keep this consistent with meson (currently using its libdir for that). Could you make those changes if you're interested?

Cheers,
Paul

Reply:
Thanks, I'll do the change ASAP.

Reply:
Looks like this line still needs to be changed to use LIBVA_DRIVER_PATH ;)

Reply:
Oops.. this is what happens when you develop on one machine and then copy-paste to another and commit (I haven't yet figured out the proper tooling for this project)... :(